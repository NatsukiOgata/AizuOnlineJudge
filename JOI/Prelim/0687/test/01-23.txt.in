75
1567 1549 1307 1629 1261 1534 1240 1351 1322 1432 1686 1591 1499 1399 1382 1315 1230 1425 1538 1277 1381 1437 1281 1568 1490 1463 1620 1317 1593 1627 1247 1485 1550 1616 1375 1273 1598 1470 1622 1215 1512 1358 1690 1386 1388 1700 1529 1660 1592 1391 1460 1609 1241 1599 1344 1585 1417 1482 1400 1456 1380 1489 1292 1442 1346 1478 1448 1580 1244 1383 1480 1371 1636 1684 1603
