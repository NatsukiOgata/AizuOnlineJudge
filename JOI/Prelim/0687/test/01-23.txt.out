65096
42621
